addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -21,8 +26,10 @@
     border-left: none;
 }
 
-.o_ChatterTopbar_leftSection {
+.o_ChatterTopbar_followers {
     display: flex;
+    flex: 1;
+    justify-content: flex-end;
```

Comment:
this rule should apply to left section of topbar, not on followers specifically.

Reply:
I don't understand why. Followers is a div containing the "follow" button and the followers counter and we want these two to be at the end of the flex. If we move the rule to the "left section" (which does not exist anymore really but I imagine you're talking of everything before the autogrow) then all the content will be align to the right which is not what we want... Am I wrong ?

Reply:
Oops, mixed my left and right. Shame it happens too frequently... 🤦

Whenever you change the XML of the right section, you likely have to adapt these rules.
IMO, this indicates this rule should be moved somewhere else.
The idea is, in LTR, not to "align right the followers", but instead "align (right the) right section"

Reply:
What do you think of just renaming "o_ChatterTopbar_followers" "o_ChatterTopbar_rightSection" ?